test/example/deployment.yaml

```diff
@@ -19,14 +31,6 @@ spec:
     spec:
       containers:
       - name: example
-        image: docker.io/library/busybox:stable
-        command:
-        - sh
-        - -c
-        - |
-          trap exit TERM
-          while true; do
-              date
-              sleep 10 &
-              wait  # interrupted quickly on signal.
-          done
+        image: docker.io/library/nginx:stable
+        ports:
+        - containerPort: 80
```

Comment:
Adding name allow setting targetPort in the service:

    - containerPort: 80
      name: example-port